src/domain/app/apollo/cache.ts

```diff
@@ -0,0 +1,74 @@
+import {
+  defaultDataIdFromObject,
+  InMemoryCache,
+  StoreObject,
+} from '@apollo/client';
+const excludeArgs =
+  (excludedArgs: string[]) =>
+  (
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    args: Record<string, any> | null
+  ) =>
+    args
+      ? Object.keys(args).filter((key: string) => !excludedArgs.includes(key))
+      : false;
```

Comment:
Thought I'd refactor this moved function to be faster with the use of Set (it was O(n*m) before) and now it should be much faster on average, and also make the code more readable at a glance by using the keyword `function` here to make the nesting of functions more readily apparent than it is when using nested arrow functions:
```suggestion

function excludeArgs(excludedArgs: string[]) {
  // eslint-disable-next-line @typescript-eslint/no-explicit-any
  return function (args: Record<string, any> | null) {
    const excludedArgsSet = new Set(excludedArgs);
    return args
      ? Object.keys(args).filter((key: string) => !excludedArgsSet.has(key))
      : false;
  };
}
```

Reply:
Looking at the code this is not used much, and is only used with ['page'] parameter so not much gain here. But, a bit of readability, and the code change does not add much complexity to the code either, so the tradeoff is okay IMHO.